OSDOCS-12183-rework: rework of changes in delete Kustomize manifest resources (#84633)
openshift/openshift-docs: modules/microshift-manifests-removal-scenario-rpm.adoc

+
[source,terminal]
----
$ sudo mkdir -p <path_of_delete_directory> <1>
----

> <1> Replace `_<path_of_delete_directory>_` with one of the following valid directory paths: `/etc/microshift/manifests.d/delete`, `/etc/microshift/manifests/delete/`, `/usr/lib/microshift/manifests.d/delete` or `/usr/lib/microshift/manifests/delete`.

Reviewer: Nit, but also out of scope - [serial/Oxford comma](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-between-items-in-a-series) needed here:
```suggestion
<1> Replace `_<path_of_delete_directory>_` with one of the following valid directory paths: `/etc/microshift/manifests.d/delete`, `/etc/microshift/manifests/delete/`, `/usr/lib/microshift/manifests.d/delete`, or `/usr/lib/microshift/manifests/delete`.
```

Author: done

. Move the manifest file into one of the `delete` subdirectories under the configured manifests directory by running the following command:
+
[source,terminal]
----
$ [sudo] mv <path_of_manifests> <path_of_delete_directory> <1>
----

> <1> Replace `_<path_of_manifests>_` with the path of the manifest to be deleted, for example, `/etc/microshift/manifests.d/010-SOME-MANIFEST`. Replace `_<path_of_delete_directory>_` with one of the following valid directory paths: `/etc/microshift/manifests.d/delete`, `/etc/microshift/manifests/delete`, `/usr/lib/microshift/manifests.d/delete` or `/usr/lib/microshift/manifests/delete`.

Reviewer: same out of scope nit about serial comma:
```suggestion
<1> Replace `_<path_of_manifests>_` with the path of the manifest to be deleted, for example, `/etc/microshift/manifests.d/010-SOME-MANIFEST`. Replace `_<path_of_delete_directory>_` with one of the following valid directory paths: `/etc/microshift/manifests.d/delete`, `/etc/microshift/manifests/delete`, `/usr/lib/microshift/manifests.d/delete`, or `/usr/lib/microshift/manifests/delete`.
```

> <1> Replace `_<path_of_manifests>_` with the path of the manifest to be deleted, for example, `/etc/microshift/manifests.d/010-SOME-MANIFEST`. Replace `_<path_of_delete_directory>_` with one of the following valid directory paths: `/etc/microshift/manifests.d/delete`, `/etc/microshift/manifests/delete`, `/usr/lib/microshift/manifests.d/delete` or `/usr/lib/microshift/manifests/delete`.

Reviewer: Out of scope suggestion for your consideration, up to you if you're interested. But in cases like this, where you are trying to use 1 callout to talk about multiple user-replaced values on the same line, it can get a little clunky and confusing. 

For these situations, we're allowed to use [description lists](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks). Here's what that might look like here:
```suggestion
+
where:

`_<path_of_manifests>_`:: Specifies the path of the manifest to be deleted, for example `/etc/microshift/manifests.d/010-SOME-MANIFEST`. 
`_<path_of_delete_directory>_`:: Specifies one of the following valid directory paths: `/etc/microshift/manifests.d/delete`, `/etc/microshift/manifests/delete`, `/usr/lib/microshift/manifests.d/delete` or `/usr/lib/microshift/manifests/delete`.
```

If you do choose to go with this, you must also delete the callout on line 25

Author: Thank you for the suggestion.  done
